build_runner/lib/src/changes/build_script_updates.dart

```diff
@@ -9,61 +9,65 @@ import 'package:build/build.dart';
 import 'package:logging/logging.dart';
 import 'package:path/path.dart' as p;
 
-import '../asset/reader.dart';
 import '../asset_graph/graph.dart';
 import '../generate/options.dart';
 
 class BuildScriptUpdates {
-  final AssetGraph _graph;
-  final String _rootPackage;
-  final DigestAssetReader _reader;
-  final _logger = new Logger('BuildScriptUpdates');
+  final Set<AssetId> _allAssetIds;
```

Comment:
This isn't very descriptive to me. How about `buildScriptAssets` or `buildScriptDependencies`?

Reply:
This is already in a `BuildScriptUpdates` class so I think that context already exists, but I did change it to `_allSources` which should make it more clear?

Reply:
`_allSources` SGTM